src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -1034,6 +1035,15 @@ bool CompilerOpenFPGA::Analyze() {
 }
 
 bool CompilerOpenFPGA::Synthesize() {
+  SCOPE_EXIT {
```

Comment:
https://en.cppreference.com/w/cpp/experimental/scope_exit

Reply:
I saw that, but experimental didn't sound like a good title to be using in production code.